craft/capctl/src/capctl.scala

```diff
@@ -47,7 +48,7 @@ class NcapctlTop(c: NcapctlTopParams)(implicit p: Parameters) extends NcapctlTop
   ioBridgeSink := imp.ioBridgeSource
 
   // create a new ports for odata
-  val ioBridgeSource = BundleBridgeSource(() => new NcapctlTopIO())
+  val ioNode = BundleBridgeSource(() => new NcapctlTopIO())
```

Comment:
I renamed `ioBridgeSource` to `ioNode` because it is convention to use`ioNode` for these IO bundle bridge nodes rocket-chip.